authors/apps/authentication/models.py

```diff
@@ -117,4 +117,17 @@ def get_short_name(self):
         """
         return self.username
 
-
+    @property
+    def token(self):
+        """ user jwt token property. """
+        return self._generate_jwt_token()
+
+    def _generate_jwt_token(self):
+        """ Method to generate user jwt token. """
+        time = datetime.now() + timedelta(days=5)
```

Comment:
Since we have a new token each time the user logs in we can set the expiry time to reflect the max time we expect them to stay in the system. 5 days may be a stretch.